README.md

```diff
@@ -77,9 +84,9 @@ rm install.sh
 sudo snap install microk8s --classic
 sudo usermod -a -G microk8s $USER
 sudo chown -f -R $USER ~/.kube
-microk8s enable ingress dns storage rbac gpu
-microk8s enable host-access:ip=10.0.1.20
-# add a line `10.0.1.20<TAB>tensorleap.local` to `/etc/hosts`
+sudo microk8s enable ingress dns storage rbac gpu
+sudo microk8s enable host-access:ip=10.0.1.20
+echo '10.0.1.20 tensorleap.local' | sudo tee -a /etc/hosts
```

Comment:
I'm not sure this this will work (because of permissions)
Unless it some hack I don't know?

Reply:
As far as I'm aware `sudo tee` is the only way to `sudo echo` into a file

Reply:
ok, cook stuff right there!